### ISSUE TITLE
Alternative text of icons

### ISSUE THREAD

**Issue by desobryc** [swiss/styleguide#249]
![alternative text icons](https://cloud.githubusercontent.com/assets/7644740/4439077/707f57e0-47b4-11e4-9120-867f28220a5c.PNG)
This is, if I understand it right, an editorial task. But it should be technically possible to add them. @zufrieden can you confirm that this is no technical issue? 
I think we should add a remark about alternative texts of icons here http://swiss.github.io/styleguide/base_layout_-_general.html#e-icons and that should be enough. Is it so ok @swiss/access4all ?
